test/issues/issues-test.js

```diff
@@ -533,4 +533,16 @@ describe("issues", function () {
             assert.same(calledWith, false);
         });
     });
+
+    describe("#1882", function () {
+        it("should use constructor name when checking deepEquality", function () {
+            class ClassWithoutProps {}
```

Comment:
`class` is an ES6 convenience over using `prototype`. This means that you can achieve the same using `prototype`.

```js
function ClassWithoutProps(){}
ClassWithoutProps.prototype.constructor = ClassWithoutProps;
```

Reply:
Ahhh, yeah let me change this...